### PR TITLE
3683 add screen and menu

### PR DESCRIPTION
**Pull request by junitvn**
https://edu-redmine.framgia.vn/issues/3683

### CODE REVIEW

**Comment by junitvn** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
-                <action android:name="android.intent.action.MAIN"/>
+                <action android:name="android.intent.action.MAIN" />
 
-                <category android:name="android.intent.category.LAUNCHER"/>
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".screen.movie_detail.MovieDetailActivity">
-        </activity>
+        <activity android:name=".screen.movie_detail.MovieDetailActivity"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag

Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]:
```suggestion
        <activity android:name=".screen.movie_detail.MovieDetailActivity"/>
```

File: app/src/main/java/com/framgia/moviedb/screen/search/SearchActivity.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class SearchActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_search);
+    }
+
+    public static Intent getIntent(Context context) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moviedb/screen/search/SearchActivity.java]: convention 2.8

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,4 +1,7 @@
 <resources>
     <string name="app_name">MovieDB</string>
     <string name="error_show_image_home">Fail to load movies. Please check your connection!</string>
+    <string name="title_activity_main">MainActivity</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: title này dùng đẻ làm gì

**Comment by DoanVanToan** [comment]
commented

**Comment by junitvn** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**